src/components/dashboard/containers/Tables/CampaignsTable.js

```diff
@@ -179,16 +195,38 @@ const getCols = ({
 			filter: false,
 			sort: true,
 			download: false,
-			customBodyRender: ({ to }) => (
-				<Tooltip
-					title={t('LABEL_VIEW')}
-					// placement='top'
-					enterDelay={1000}
-				>
-					<RRIconButton to={to} variant='contained' aria-label='preview'>
-						<Visibility color='primary' />
-					</RRIconButton>
-				</Tooltip>
+			customBodyRender: ({ side, id, humanFriendlyName }) => (
+				<React.Fragment>
+					<Tooltip
+						title={t('LABEL_VIEW')}
+						// placement='top'
+						enterDelay={1000}
+					>
+						<RRIconButton
+							to={`/dashboard/${side}/Campaign/${id}`}
+							variant='contained'
+							aria-label='preview'
+						>
+							<Visibility color='primary' />
+						</RRIconButton>
+					</Tooltip>
+					{humanFriendlyName === 'Closed' ||
```

Comment:
again should be `(humanFriendlyName === 'Closed' || humanFriendlyName === 'Completed') && ...`